test/github-issues/3534/issue-3534.ts

```diff
@@ -0,0 +1,32 @@
+import 'reflect-metadata';
+import {expect} from 'chai';
+import { Connection, PromiseUtils } from '../../../src';
+import { Foo } from './entity/Foo';
+import { closeTestingConnections, createTestingConnections } from '../../utils/test-utils';
+
+describe('github issues > #3534: store regexp', () => {
+    let connections: Connection[];
+    before(async () => {
+        connections = await createTestingConnections({
+            entities: [__dirname + '/entity/*{.js,.ts}'],
+            enabledDrivers: ['postgres'],
```

Comment:
can you please enable all drivers? since we don't have anything postgres specific